server/src/main/java/org/apache/gravitino/server/web/rest/TableOperations.java

```diff
@@ -84,6 +89,18 @@ public Response listTables(
           () -> {
             Namespace tableNS = NamespaceUtil.ofTable(metalake, catalog, schema);
             NameIdentifier[] idents = dispatcher.listTables(tableNS);
+            idents =
+                MetadataFilterHelper.filterByExpression(
+                    metalake,
+                    "( (METALAKE::USE_CATALOG || CATALOG::USE_CATALOG) && "
+                        + "(METALAKE::USE_SCHEMA || CATALOG::USE_SCHEMA ||SCHEMA::USE_SCHEMA) &&"
+                        + " (METALAKE::SELECT_TABLE || CATALOG::SELECT_TABLE || "
+                        + "SCHEMA::SELECT_TABLE || TABLE::SELECT_TABLE || "
```

Comment:
Please use any expression like ANY(SELECT_TABLE,METALAKE,CATALOG,SCHEMA,TABLE)

Please refer to https://github.com/apache/gravitino/pull/7577/files